tests/behat/Makefile

```diff
@@ -1,40 +1,42 @@
-test-setup: setup-modules setup-config
+# Allow for overriding the drush command. Handy if running drush in a
+# container.
+DRUSH ?= drush
 
-# Install a test site.
-install-site:
-	drush site-install ding2
+test-setup: setup-modules setup-config
 
 # Setup test modules on site.
 setup-modules:
 	# We use the connie test provider for testing. Enable it and
 	# disable the default fbs.
-	drush en -y connie
-	drush dis -y fbs
+	$(DRUSH) en -y connie
+	$(DRUSH) dis -y fbs
 
 	# Enable mocking openlist provider.
-	drush en -y connie_openlist
+	# Currently disabled as the API has changed and Connie haven't been updated.
+	# $(DRUSH) en -y connie_openlist
```

Comment:
does this still hold true? connie_openlist is updated as part of this PR, is it not?

Reply:
No it's not. There was no time to try to reverse engineer openlists so-called API. Runs against `https://test.openlist.ddbcms.dk/`. At some time it might get fixed to running Openlist in a separate container instead.